modules/ROOT/pages/fd-reuse-config.adoc

```diff
@@ -6,11 +6,13 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can reuse a connection in a Flow Designer project instead of adding your authentication information manually. After a reusable connection is created, it is available for all Flow Designer projects in your business group. Changing your resuable configuration is can be done anytime. 
+You can reuse a connection in a Flow Designer project instead of manually adding your authentication information.
+After a reusable connection is created, it is available for all Flow Designer projects in your business group.
+You can change your reusable configuration at anytime.
 
 You can reuse connections:
 
-* If you are a system administrator, you can enable non-administrators to connect to your system using your authentication. 
+* If you are a system administrator, you can enable non-administrators to connect to your system using your authentication.
```

Comment:
"authentication" > "credentials"?

Reply:
Note that the step at Line 28, which is hidden in my copy of this file and cannot be edited directly, violates the one-sentence-per-list-item style guideline. I suggest reformatting it as follows:

. To make this connection available for users in your business group, leave the *share this connection with my business group* checkbox selected. 
To maintain security, only you as the creator of a shared connection can view your authentication credentials.

Reply:
The rest of the FD doc says authentication. That's why I kept it. I would have to change it globally.